_content/av/buddhist-mythology_sujato.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Buddhist Mythology: The Sacred and the Profane"
+authors:
+  - sujato
+subcat: course
+external_url: "https://discourse.suttacentral.net/t/bhante-sujato-buddhist-mythology-course-sydney-2019/13204"
+drive_links:
+  - "https://drive.google.com/file/d/1ynz8Zou66pCMk8Ujz0_ZWYYlFESQnEnv/view?usp=drive_link"
+  - "https://drive.google.com/file/d/1nDK2aaeWYe8-mHaI4NtC0Lz1AjpKszpN/view?usp=drive_link"
+  - "https://drive.google.com/file/d/1J9wb-j6u3pIq9RdDZyCGDZbLY4qo-AtK/view?usp=drive_link"
+course: rebirth-stories
+tags:
+  - mythology
+year: 2019
+month: Jun
+minutes: 322
+---
+
+This course explores various modern interpretations of myth to shed light on Buddhist stories, emphasizing the importance of mythology in understanding how we, as followers of the Buddha's teachings, can learn from those who came before us. Bhante Sujato explains how, using Buddhist mythology, Buddhist practitioners have wrestled with the challenge of reconciling the pure and exalted teachings of the Buddha.
+
+The fourth epsisode in this course is not required. However, all four epsiodes can be found at https://discourse.suttacentral.net/t/ajahn-sujato-buddhist-mythology-the-sacred-and-the-profane/13132/1
```

Comment:
My style is to generally not have naked links like this.  Use `[markdown format](https://www.markdownguide.org/basic-syntax/#links)` to give the link a nice title.

Also, you didn't mention the most important part which is that this link is to the YouTube videos